_reports/2022-00005 - Exposed BACnet Devices.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: report
+has_casefile: true
+date: 2023-05-20
+title: Exposed BACnet Devices
+case: DIVD-2022-00005
+author: Gerard Janssen
+lead: Ruben Uithol
+csirt: []
+researchers:
+- Artur Miron
+- Patrick Hulshof
+- Ruben Uithol
+excerpt: Exposed BACnet ports
+start: 2022-01-29
+end:
+timeline:
```

Comment:
Dit werkt inderdaad niet, omdat ik dit niet in de www site report template heb "ingebakken".
Je zou wel de HTML coce van de timeline kunnen kopieeren in de HTML hieronder.

Reply:
ah zoals ik ook deed in Forrestry dus